docker/supabase/volumes/db/init/04-initiate-tables.sql

```diff
@@ -0,0 +1,14 @@
+CREATE extension IF NOT EXISTS citext  WITH SCHEMA extensions;
+
+CREATE TABLE IF NOT EXISTS telescope_profiles (
+  id text PRIMARY KEY,
+  registered_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+  updated_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+  feeds text[] NOT NULL,
```

Comment:
I wonder if we should move this to a separate table (follow-up PR) with a foreign key relationship on the `id` for the user?  The idea of a user having feeds is something that could change, it's not really part of your profile.